value_sensors/remote_sensor.lua

```diff
@@ -1,11 +1,13 @@
 require "template"
 
 RemoteSensor = {}
-function RemoteSensor.new(name, line, caption, color)
-    local sensor = ValueSensor.new("remote_sensor_" .. name)
-    sensor["line"] = line
-    sensor["display_name"] = caption
-    sensor["color"] = color
+function RemoteSensor.new(sensor_data)
+    local sensor = ValueSensor.new("remote_sensor_" .. sensor_data.name)
+
+    sensor["mod_name"] = sensor_data.mod_name
+    sensor["line"] = sensor_data.text
+    sensor["display_name"] = "[" .. sensor_data.mod_name .. "]" .. sensor_data.caption
```

Comment:
Probably prefer the table-aware localized string concatenation, i.e., `{"", "[", sensor_data.mod_name, "] ", sensor_data.caption}` -- this should allow consumers to have their mod display name localized.